onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorElementWise.cpp

```diff
@@ -499,7 +499,7 @@ class DmlOperatorElementwiseQLinear : public DmlOperator
 public:
     DmlOperatorElementwiseQLinear(const MLOperatorKernelCreationContext& kernelInfo) : DmlOperator(kernelInfo)
     {
-        ML_CHECK_VALID_ARGUMENT(kernelInfo.GetInputCount() == 3);
+        ML_CHECK_VALID_ARGUMENT(kernelInfo.GetInputCount() == 3); // TODO: Can be 2 inputs, since x_zero_point is optional.
```

Comment:
If we don't support optional `x_zero_point` yet, we should add a query for it (e.g. like the `Attention` op is doing) instead of hard crashing.

Reply:
See updates, I added a mechanism to construct a zeropointtensor if one isn't defined.